task/deprecated-image-check/0.3/deprecated-image-check.yaml

```diff
@@ -60,7 +63,9 @@ spec:
           value: $(params.POLICY_NAMESPACE)
       script: |
         #!/usr/bin/env sh
+        set -euo pipefail
         source /utils.sh
+        trap cleanup ERR
```

Comment:
I might be undestanding this wrongly, but why this script doesn't include
` trap 'echo "Line:$LINENO"' ERR`
I see pipe in script below.

Reply:
That cleanup is defined in https://github.com/redhat-appstudio/hacbs-test/pull/137/files#diff-b17e8443b3f0dd1a2f1701af585a3a6222f2631b717c934c6fede3c8b16837ecR161 we import and call it.